src/locales/en-US.json

```diff
@@ -1072,7 +1072,8 @@
     "username": "Username",
     "alert": {
       "unauthorizedMessage": "User unauthorized",
-      "message": "Invalid username or password"
+      "message": "Invalid username or password",
+      "refreshMessage": "If you are failed to login, refresh the browser."
```

Comment:
So any failed login we display this message..? 
The hostname is rarely changed, do we really want that edge case to influence this? We already warn the user

Reply:
Yes.
It is rarely encountered. This happened to me last day. Then I refreshed the BMC GUI webpage and logged in with the service credentials, it worked.  Any future occurrence of this issue will trigger this toast message.

Reply:
I don't think we should add this message

Reply:
> Makes it seem like a refresh will fix.. but I think for 99.9% of the time, a refresh will not help. Is this displayed any time a "Invalid username or password" is hit?

No, It won't.

Reply:
It will not display if we have "Invalid username or password", But yes, we can consider not showing this message if it is null, And also, we would check why it is null after changing hostname.

Reply:
Removed this message as we are showing a message before making the change in hostname.